docs/src/snap/tutorial/backing-up.md

```diff
@@ -0,0 +1,295 @@
+
+```bash
+sudo k8s helm repo add minio https://charts.min.io
+sudo k8s kubectl create namespace velero
+
+# TOY MODE
+sudo k8s helm install -n velero \
+--set resources.requests.memory=512Mi \
+--set replicas=1 \
+--set persistence.enabled=false \
+--set mode=standalone \
+--set buckets[0].name=velero,buckets[0].policy=none,buckets[0].purge=false \
+--generate-name minio/minio
+
+# PRODUCTION MODE
+sudo k8s helm install -n velero \
+--set buckets[0].name=velero,buckets[0].policy=none,buckets[0].purge=false \
+--generate-name minio/minio
+```
+
+### Create a demo workload
+
+The workload we will demonstrate the backup with is an NGINX deployment and a
+corresponding service under the `workloads` namespace. Create this setup with:
+
+```bash
+sudo k8s kubectl create namespace workloads
+sudo k8s kubectl create deployment nginx -n workloads --image nginx
+sudo k8s kubectl expose deployment nginx -n workloads --port 80
+```
+
+
+## Installing Velero
+
+To install Velero we get the a binary from the [releases page on
+github][releases] and place it in our `PATH`. In this case we install the
+v1.14.1 Linux binary for AMD64 under `/usr/local/bin`:
+
+```bash
+wget https://github.com/vmware-tanzu/velero/releases/download/v1.14.0/velero-v1.14.0-linux-amd64.tar.gz 
+tar -xzf velero-v1.14.0-linux-amd64.tar.gz
+chmod +x velero-v1.14.0-linux-amd64/velero
+sudo chown root:root velero-v1.14.0-linux-amd64/velero
+sudo mv velero-v1.14.0-linux-amd64/velero /usr/local/bin/velero
+```
+
+Before installing Velero, we export the kubeconfig file from MicroK8s.
+
+```bash
+mkdir -p $HOME/.kube
+sudo k8s kubectl config view --raw > $HOME/.kube/config
+```
+
+We also export the MinIO credentials so we can feed them to Velero.
+
+```bash
+ACCESS_KEY=$(sudo k8s kubectl -n velero get secret -l app=minio -o jsonpath="{.items[0].data.rootUser}" | base64 --decode)
+SECRET_KEY=$(sudo k8s kubectl -n velero get secret -l app=minio -o jsonpath="{.items[0].data.rootPassword}" | base64 --decode)
+SERVICE=$(sudo k8s kubectl -n velero get svc -l app=minio -o jsonpath="{.items[0].metadata.name}")
+cat <<EOF > credentials-velero
+[default]
+    aws_access_key_id=${ACCESS_KEY}
+    aws_secret_access_key=${SECRET_KEY}
+EOF
+```
+
+We are now ready to install Velero, with the aws [plugin
+matching][aws-plugin-matching] the velero release:
+
+```bash
+SERVICE_URL="http://${SERVICE}.velero.svc:9000"
+BUCKET=velero
+REGION=minio
+velero install \
+    --provider aws \
+    --plugins velero/velero-plugin-for-aws:v1.10.0 \
+    --bucket $BUCKET \
+    --backup-location-config region=$REGION,s3ForcePathStyle="true",s3Url=$SERVICE_URL \
+    --snapshot-location-config region=$REGION \
+    --secret-file ./credentials-velero
+```
+
+
+## Backup workloads
+
+To backup the `workloads` namespace we use the `--include-namespaces` argument:
+ 
+```bash
+velero backup create workloads-backup --include-namespaces=workloads
+```
+
+> ⓘ **Note:** Please, consult the [official Velero
```

Comment:
```suggestion
> **Note:** Please, consult the [official Velero
```

I like the aesthetic but let's remove it to keep consistent with the rest of the docs here. (Example: https://github.com/canonical/k8s-snap/blob/f0130a9f2399367a6e4a18457737da0a3a6cca4c/docs/src/snap/tutorial/kubectl.md?plain=1#L32)